<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettingsPage</class>
 <widget class="QWidget" name="AppearanceSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="languageComboBox">
       <property name="toolTip">
        <string>Set the application language. Requires restart!</string>
       </property>
       <item>
        <property name="text">
         <string>&lt;Untranslated&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;System Default&gt;</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Widget style:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="styleComboBox">
       <property name="toolTip">
        <string>Set application style</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="iconthemeLabel">
       <property name="text">
        <string>Fallback icon theme:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="iconThemeComboBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Icon theme to use for icons that are not found in the current system theme. Requires the selected theme to be installed either system-wide, or as part of the Quassel installation.&lt;/p&gt;&lt;p&gt;Supported themes are Breeze, Breeze Dark and Oxygen, all of KDE fame.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="overrideSystemIconTheme">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If enabled, uses the selected fallback icon theme instead of the configured system theme for all icons. Recommended if you want Quassel to have a consistent look-and-feel.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Override system theme</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">/UiStyle/Icons/OverrideSystemTheme</string>
       </property>
       <property name="defaultValue" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="useCustomStyleSheet">
       <property name="text">
        <string>Use custom stylesheet</string>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">/UiStyle/UseCustomStyleSheet</string>
       </property>
       <property name="defaultValue" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customStyleSheetPath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">/UiStyle/CustomStyleSheetPath</string>
       </property>
       <property name="defaultValue" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="chooseStyleSheet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="useSystemTrayIcon">
       <property name="text">
        <string>Show system tray icon</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">UseSystemTrayIcon</string>
       </property>
       <property name="defaultValue" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="invertSystrayColors">
       <property name="text">
        <string>Invert brightness</string>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">/UiStyle/Icons/InvertTray</string>
       </property>
       <property name="defaultValue" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="minimizeOnClose">
       <property name="text">
        <string>Hide to tray on close button</string>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">MinimizeOnClose</string>
       </property>
       <property name="defaultValue" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Message Redirection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,1">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>User Notices:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="userNoticesInDefaultBuffer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="userNoticesInStatusBuffer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="userNoticesInCurrentBuffer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Server Notices:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="serverNoticesInDefaultBuffer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Default Target</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Status Window</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Current Chat</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Errors:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="serverNoticesInStatusBuffer">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="serverNoticesInCurrentBuffer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="errorMsgsInDefaultBuffer">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="errorMsgsInStatusBuffer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="errorMsgsInCurrentBuffer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>languageComboBox</tabstop>
  <tabstop>styleComboBox</tabstop>
  <tabstop>iconThemeComboBox</tabstop>
  <tabstop>overrideSystemIconTheme</tabstop>
  <tabstop>useCustomStyleSheet</tabstop>
  <tabstop>chooseStyleSheet</tabstop>
  <tabstop>customStyleSheetPath</tabstop>
  <tabstop>useSystemTrayIcon</tabstop>
  <tabstop>minimizeOnClose</tabstop>
  <tabstop>invertSystrayColors</tabstop>
  <tabstop>userNoticesInDefaultBuffer</tabstop>
  <tabstop>userNoticesInStatusBuffer</tabstop>
  <tabstop>userNoticesInCurrentBuffer</tabstop>
  <tabstop>serverNoticesInDefaultBuffer</tabstop>
  <tabstop>serverNoticesInStatusBuffer</tabstop>
  <tabstop>serverNoticesInCurrentBuffer</tabstop>
  <tabstop>errorMsgsInDefaultBuffer</tabstop>
  <tabstop>errorMsgsInStatusBuffer</tabstop>
  <tabstop>errorMsgsInCurrentBuffer</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>useCustomStyleSheet</sender>
   <signal>toggled(bool)</signal>
   <receiver>customStyleSheetPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomStyleSheet</sender>
   <signal>toggled(bool)</signal>
   <receiver>chooseStyleSheet</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useSystemTrayIcon</sender>
   <signal>toggled(bool)</signal>
   <receiver>minimizeOnClose</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useSystemTrayIcon</sender>
   <signal>toggled(bool)</signal>
   <receiver>invertSystrayColors</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
